jsonstore.go

```diff
@@ -61,23 +63,24 @@ func Open(filename string) (*JSONStore, error) {
 func Save(ks *JSONStore, filename string) (err error) {
 	ks.RLock()
 	defer ks.RUnlock()
-
 	toSave := make(map[string]string)
 	for key := range ks.Data {
 		toSave[key] = string(ks.Data[key])
 	}
-	b, err := json.MarshalIndent(toSave, "", " ")
+	var w io.Writer
+	f, err := os.OpenFile(filename, os.O_RDWR, 0644)
```

Comment:
I totally forgot about new files! Thanks for finding that. I'll fix it and resubmit soon~ish